vault/login_mfa.go

```diff
@@ -2013,11 +2015,29 @@ func (c *Core) validateTOTP(ctx context.Context, creds []string, entityMethodSec
 
 	usedName := fmt.Sprintf("%s_%s", configID, creds[0])
 
-	_, ok := c.loginMFABackend.usedCodes.Get(usedName)
+	_, ok := usedCodes.Get(usedName)
 	if ok {
 		return fmt.Errorf("code already used; new code is available in %v seconds", totpSecret.Period)
 	}
 
+	numAttempts, _ := usedCodes.Get(configID)
+	if numAttempts == nil {
+		usedCodes.Set(configID, int64(1), defaultMFAAuthResponseTTL)
+	} else {
+		err := usedCodes.Increment(configID, 1)
```

Comment:
I assume the TTL remains unaffected by this call?

Reply:
yes